easybuild/easyconfigs/t/ToFu/ToFu-1.3.17-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'Bundle'
+
+name = 'ToFu'
+version = '1.3.17'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ToFuProject/tofu'
+description = """Tomography for Fusion."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('IPython', '5.7.0', versionsuffix),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('MDSplus', '7.46.1'),
+]
+
+# raise an error when auto-downloading of dependency packages is detected
+exts_download_dep_fail = True
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [
+    ('docutils', '0.14', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/docutils'],
+        'checksums': ['51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274'],
+    }),
+    ('Polygon2', '2.0.8', {
+        'modulename': 'Polygon',
+        'source_urls': ['https://pypi.python.org/packages/source/p/polygon2'],
+        'checksums': ['4ae3b027483daf1e967a96bceca18481d57cfa30480c579c3bac32f5c3fe9c84'],
+    }),
+    ('zope.interface', '4.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/z/zope.interface'],
+        'checksums': ['57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c'],
+    }),
+    ('DateTime', '4.2', {
+        'modulename': 'DateTime',
+        'source_urls': ['https://pypi.python.org/packages/source/d/datetime'],
+        'checksums': ['b3d91011caff312274a629b35abd6eb67038f8f92423ad3d9e071c0bf8823226'],
+    }),
+    ('tofu', version, {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tofu'],
+        'checksums': ['263afa286cda0ae70272268548bf344a4dd68a76f4a3de04dfd57deffdfe4169'],
+    }),
+]
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+sanity_check_paths = {
+    'files': ['bin/easy_install-%(pyshortver)s'],
```

Comment:
@SimonPinches I wouldn't check for this file, the fact that it's there is a side-effect of installing with `setuptools`, it's not actually provided by `ToFu` itself...

We may switch to installing Python packages with `pip` under the hood in EasyBuild 4.0 (which is not for soon, but it doesn't hurt to keep that in mind).

Just checking for `lib/python%(pyshortver)s/site-packages` directory + the import check is sufficient imho...